Description: Fix regular expression
 >>> import os
 >>> import re
 >>> import string
 >>> import sys
 >>> netsnmp_libs="-Wl,-z,relro -Wl,-z,now -L/usr/lib/x86_64-linux-gnu -lnetsnmp -lcrypto -lm"
 >>> print netsnmp_libs
 -Wl,-z,relro -Wl,-z,now -L/usr/lib/x86_64-linux-gnu -lnetsnmp -lcrypto -lm
 >>> re.findall(r"-l(\S+)", netsnmp_libs)
 ['inux-gnu', 'netsnmp', 'crypto', 'm']
 .
  Just inserting space will fix this.
 >>> re.findall(r" -l(\S+)", netsnmp_libs)
 ['netsnmp', 'crypto', 'm']
Author: Hideki Yamane <henrich@debian.org>
Reviewed-by: Craig Small <csmall@debian.org>
LAst-Update: 2019-02-07
--- a/python/setup.py
+++ b/python/setup.py
@@ -17,14 +17,14 @@
     netsnmp_libs = os.popen(basedir+'/net-snmp-config --libs').read()
     libdir = os.popen(basedir+'/net-snmp-config --build-lib-dirs '+basedir).read()
     incdir = os.popen(basedir+'/net-snmp-config --build-includes '+basedir).read() + " " + os.popen(basedir+'/net-snmp-config --base-cflags '+basedir).read()
-    libs = re.findall(r"-l(\S+)", netsnmp_libs)
+    libs = re.findall(r" -l(\S+)", netsnmp_libs)
     libdirs = re.findall(r"-L(\S+)", libdir)
     incdirs = re.findall(r"-I(\S+)", incdir)
 else:
     netsnmp_libs = os.popen('net-snmp-config --libs').read()
     libdirs = re.findall(r"-L(\S+)", netsnmp_libs)
     incdirs = []
-    libs = re.findall(r"-l(\S+)", netsnmp_libs)
+    libs = re.findall(r" -l(\S+)", netsnmp_libs)
 
 setup(
     name="netsnmp-python", version="1.0a1",
